nautobot/docs/rest-api/authentication.md

```diff
@@ -40,6 +40,8 @@ $ curl http://nautobot/api/dcim/sites/
 
 ## Initial Token Provisioning
 
+_Added in version [1.3.0](../release-notes/version-1.3.md/#rest-api-token-provisioning-1374)_
```

Comment:
Do we have a housekeeping item open for going through and adding this sort of note throughout?

Reply:
https://github.com/nautobot/nautobot/issues/1778